### ISSUE TITLE
TSVGIconImage fails to draw if it is aligned on the sides

### ISSUE THREAD

**Issue by DanielAraujo-dac** [EtheaDev/SVGIconImageList#75]
## Error
When we align TSVGIconImage to the left or right and its width is small, the image is not drawn on the screen.

## Video
[Record_2020_08_25_20_35_58_943.zip](https://github.com/EtheaDev/SVGIconImageList/files/5126908/Record_2020_08_25_20_35_58_943.zip)

**Comment by pyscripter** [comment]
Could you please post here the text of your svg?

**Comment by pyscripter** [comment]
@Carlo Have you checked this?  I cannot reproduce it.

**Comment by carloBarazzetta** [comment]
Not yet...

**Comment by DanielAraujo-dac** [comment]
> Could you post your svg text here?

I'm using the same DEMO SVG made available in the repository.

Demo: `.. \ Demo \ D10_4 \ SVGIconImageDemoFMX.dproj`
SVGText
``` svg
<svg version = "1" xmlns = "http://www.w3.org/2000/svg" viewBox = "0 0 48 48" enable-background = "new 0 0 48 48">
<path fill = "# 4CAF50 "d =" M21.2,44.8l-18-18c-1.6-1.6-1.6-4.1,0-5.7l18-18c1.6-1.6,4.1-1.6,5.7,0l18,18c1,6,1.6,1.6 , 4.1,0,5.7l-18,18 C25.3,46.4,22.7,46.4,21.2,44.8z "/>
<g fill =" # FFEB3B "> <polygon points =" 24,33.4 17,25 31, 25 "/> <rect x =" 22 "y =" 14.8 "width =" 4 "height =" 12.3 "/> </g> </svg>
```

**Comment by pyscripter** [comment]
I just noticed that the issue is on the FMX side.

**Comment by carloBarazzetta** [comment]
To fix this problem change this method, removing the "if" inside it:
procedure TSVGIconFixedBitmapItem.SetSize(const AValue: Integer);
begin
  FSize := AValue;
  DrawSVGIcon;
end;

In the next version I will fix it.

**Comment by carloBarazzetta** [comment]
Fixed in 2.2.1 version
